src/string_util.h

```diff
@@ -39,6 +39,11 @@ inline std::string StrCat(Args&&... args)
 void ReplaceAll(std::string* str, const std::string& from,
                 const std::string& to);
 
-} // end namespace benchmark
+std::string generateInstanceName(const std::string& name, int arg_count,
```

Comment:
this isn't really a string util, it should be local to the benchmark code.

it could even be an instance method, i think...